### DISCUSSION TITLE
Show Month/Day or Weekday/Day on 2 lines instead of one

### DISCUSSION THREAD

**Discussion by kaizersoje** [totaldebug/atomic-calendar-revive#1188] [Feature Requests] [answered]
Before upgrading to v9.0.0, the card would show the month and date in 2 separate lines.

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/54566217/125998aa-01ef-49f3-ac48-e0fb96498308)

However in v9.0.0, it shows up on one line.

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/54566217/5e6e4a2c-8e05-439b-8f64-3b5068a38cbd)

Is it possible to add the capability to show in 2 lines? Having in 2 lines is quite benefical on smaller screens so that the event titles dont wrap to multiple lines.

**Comment by marksie1988** [comment, marked as answer]
This can be done with the following code:

```
card_mod:
  style: |
    .event-left{
      display: flex;
      flex-direction: column !important;
      align-items: center;
    }
```

**Comment by kaizersoje** [comment]
Thanks @marksie1988
